profiles/CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
```

Comment:
Excellent format. We can follow the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) for our osrm-backend release too.